crates/cheatcodes/src/test/assert.rs

```diff
@@ -447,6 +448,13 @@ impl_assertions! {
     (assertApproxEqRelDecimal_2Call, assertApproxEqRelDecimal_3Call),
 }
 
+impl Cheatcode for containsCall {
```

Comment:
Please move this and the `Vm` Solidity definition to the respective file and group